README.md

```diff
@@ -85,19 +86,4 @@ module "postgresql" {
 | <a name="output_password"></a> [password](#output\_password) | n/a |
 | <a name="output_resource_group_location"></a> [resource\_group\_location](#output\_resource\_group\_location) | n/a |
 | <a name="output_resource_group_name"></a> [resource\_group\_name](#output\_resource\_group\_name) | n/a |
-<!-- END OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
-
-## Contributing
-
-We use pre-commit hooks for validating the terraform format and maintaining the documentation automatically.
-Install it with:
-
-```shell
-$ brew install pre-commit terraform-docs
-$ pre-commit install
-```
-
-If you add a new hook make sure to run it against all files:
-```shell
-$ pre-commit run --all-files
-```
```

Comment:
this has deleted the contributing section?

Reply:
Accidental